superset-frontend/cypress-base/cypress/e2e/dashboard/nativeFilters.test.ts

```diff
@@ -312,15 +312,15 @@ describe('Native filters', () => {
   });
 
   describe('Nativefilters initial state not required', () => {
```

Comment:
instead of skipping each test, i believe if you did a `describe.skip()` you'll skip all tests in the describe block